packages/react/src/components/AccountSettings/DeleteUser/defaultComponents.tsx

```diff
@@ -0,0 +1,55 @@
+import { translate } from '@aws-amplify/ui';
+import React from 'react';
+
+import { Card, Flex, Text, Button, Alert } from '../../../primitives';
+import { AccountSettingsError, AccountSettingsSubmitButton } from '../types';
+import { DeleteUserWarningProps } from './types';
+
+export const DefaultSubmitButton: AccountSettingsSubmitButton = ({
+  children,
+  ...rest
+}) => {
+  return <Button {...rest}>{children}</Button>;
+};
+
+export const DefaultWarning = ({
+  onCancel,
+  onConfirmDelete,
+  isInProgress,
+}: DeleteUserWarningProps): JSX.Element => {
+  // translations
+  // TODO: consolodiate translations to accountSettingsTextUtil
```

Comment:
Will this be done in a follow-up PR?

Reply:
Yes — it will be a launch requirement item as well. Ill create a ticket to track this too.